### DISCUSSION TITLE
Is Virtua Fighter 5 Final Showdow Supported?

### DISCUSSION THREAD

**Discussion by irodinnovations** [nirgoren/Eddienput#24] [Technical support]
Hello,

Anyone tested VF5FS?  I'm running it on my PC via RPCS3, but can't get any of my recordings to load.  I'm probably building it all wrong since I've only been tinkering Eddienput for about an hour or so.  A little help on how to get started or some simple sample code would greatly be appreciated.  I've tried looking at the other recording samples as a reference, but the console always gives me an error when I try to load a file.  Thanks!

**Comment by KuukuuOmoi** [comment]
can you screen shot the error and show the code you're using
